src/wp-includes/revision.php

```diff
@@ -450,6 +497,9 @@ function wp_restore_post_revision( $revision, $fields = null ) {
 	// Update last edit user.
 	update_post_meta( $post_id, '_edit_last', get_current_user_id() );
 
+	// Restore any revisioned meta fields.
+	wp_restore_post_revision_meta( $post_id, $revision['ID'] );
```

Comment:
Could we do this 

```
add_action( 'wp_restore_post_revision', 'wp_restore_post_revision_meta', 10, 2 );
```

Reply:
good idea, let me check.

Reply:
Great! Added in [`95508d9` (#4859)](https://github.com/WordPress/wordpress-develop/pull/4859/commits/95508d959d69e6be174c922c8308dac51077b7ff)

Reply:
@adamsilverstein Has this been addressed?

Reply:
yes, this is added